### DISCUSSION TITLE
Hardcoding Group Type and the GenMapper metric

### DISCUSSION THREAD

**Discussion by yorkaspacher** [DiscipleTools/disciple-tools-theme#1596] [Q&A]
@corsacca Similar to two other discussions — Group Type options have changed, and Group generations genmapping doesn't work, just spins. Can this plugin be adjusted to accommodate other Group Type options besides the default?

<img width="1345" alt="image" src="https://user-images.githubusercontent.com/47872338/155149268-8a88b63b-a964-42aa-bbfa-43e98935b78c.png">

**Comment by chrisanubhav** [comment]
@corsacca - Adding further details to the above question.

If we add the custom Group Type and we are going to Metrics => Gen Mapper => Groups, the above code is not working for the new Group type, to work for the new Group Type and Groups Generation Tree charts. How can we fix the issue for this?

disciple-tools-genmapper-master/includes/charts/genmapper.js: 
Function getFieldValueForRadioType()
Line no 476

**Reply by corsacca** [reply]
That looks like a bug. Created issue: https://github.com/DiscipleTools/disciple-tools-genmapper/issues/12
It looks like it is not finding the correct template and should default to "group" or something.

**Reply by corsacca** [reply]
@chrisanubhav You help looking into the issue would be appreciated

**Reply by corsacca** [reply]
@yorkaspacher @chrisanubhav,  we weren't able to reproduce the error now. Maybe it was fixed it 1.5.4. Would you be able to confirm?

**Comment by corsacca** [comment]
I think the underlying issue was found and fixed in https://github.com/DiscipleTools/disciple-tools-genmapper/releases/tag/1.7.0
